### PR TITLE
[ur] fix minor coverity issues

### PR DESCRIPTION
**Pull request by DamianDuy**
- fix copy instead of move issue

### CODE REVIEW

**Comment by PatKamin** [comment]
Update template file `valddi.cpp.mako`.

File: source/loader/layers/validation/ur_valddi.cpp [hunk 1]
```diff
@@ -3613,7 +3613,8 @@ __urdlllocal ur_result_t UR_APICALL urEnqueueMemBufferReadRect(
             return UR_RESULT_ERROR_INVALID_SIZE;
         }
 
-        if (bufferSlicePitch != 0 && bufferSlicePitch % bufferRowPitch != 0) {
```
[the remark this answers is not in the document]
Comment (DamianDuy) [reply source/loader/layers/validation/ur_valddi.cpp]: Done.

**Review by pbalcer — APPROVED** [review]

File: test/conformance/source/environment.cpp [hunk 2]
```diff
@@ -197,7 +197,8 @@ KernelsEnvironment *KernelsEnvironment::instance = nullptr;
 KernelsEnvironment::KernelsEnvironment(int argc, char **argv,
                                        std::string kernels_default_dir)
     : DevicesEnvironment(argc, argv),
-      kernel_options(parseKernelOptions(argc, argv, kernels_default_dir)) {
+      kernel_options(
+          parseKernelOptions(argc, argv, std::move(kernels_default_dir))) {
```
Comment (igchor) [inline test/conformance/source/environment.cpp:201]: nit: What coverity issue does it fix exactly? I think a better fix would be to change the parameter type to `const std::string & kernels_default_dir` - it shows the intention better

Comment (DamianDuy) [reply test/conformance/source/environment.cpp]: It is a ```copy_instead_of_move``` issue. Done.

**Review by igchor — APPROVED** [review]

**Pull request merged**